packages/builder/cypress/integration/templates/HR/jobApplicationTracker.spec.js

```diff
@@ -8,13 +8,13 @@ filterTests(["all"], () => {
     before(() => {
         cy.login()
         cy.deleteApp(templateName)
-        cy.visit(`${Cypress.config().baseUrl}/builder`, {
+        // Template navigation
+        cy.visit(`${Cypress.config().baseUrl}/builder/portal/apps/templates`, {
             onBeforeLoad(win) {
                 cy.stub(win, 'open')
             }
         })
         cy.wait(2000)
```

Comment:
It could be worth checkout out timeouts instead adding constant wait times, would mean tests will complete faster as the system performs well and vice versa, giving us a good indication of any sluggishness at times. 

https://stackoverflow.com/questions/60063879/how-to-wait-for-element-to-be-visible

If we started using these a bit more across the board we might be able to bring the build times down quite a bit

Reply:
I agree, I have done this in a couple of places across the tests. There will be a refactor including these timeouts. They will be particularly useful for the nightly smoke build since it has the most issues with timing.